client/src/app/component/header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {NavLink} from "react-router-dom";
+import clsx from "clsx";
+import styles from "src/app/component/header/Header.module.scss";
+import btnStyles from "src/app/component/button/Button.module.scss";
+
+const MAIN_PATH = "/";
+
+/**
+ * Header with links
```

Comment:
"With links" -- this info must be incapsilated. Useless in comment

Reply:
corrected